compiler/compiler_test.go

```diff
@@ -56,6 +56,7 @@ func TestCompiler(t *testing.T) {
 		{"float.go", "", ""},
 		{"interface.go", "", ""},
 		{"func.go", "", "coroutines"},
+		{"defer.go", "", ""},
```

Comment:
Can we maybe do this on a target that can actually run recover so that we know that the code gen is actually working?

Reply:
Good idea.